packages/auth/ios/RNFBAuth/RNFBAuthModule.m

```diff
@@ -55,7 +55,7 @@
 static __strong NSMutableDictionary *emulatorConfigs;
 // Used for caching credentials between method calls.
 static __strong NSMutableDictionary<NSString *, FIRAuthCredential *> *credentials;
-#if !TARGET_OS_TV
+#if TARGET_OS_IOS
```

Comment:
all the multifactor stuff is guarded this way, example: https://github.com/firebase/firebase-ios-sdk/blob/ae2554220afb03a5d6fc389c28a86061a12c7caa/FirebaseAuth/Sources/ObjC/FIRMultiFactorConstants.m#L18